Marlin/Marlin_main.cpp

```diff
@@ -1165,22 +1176,109 @@ bool code_has_value() {
   return NUMERIC(c);
 }
 
-float code_value() {
+float code_value_float() {
   float ret;
-  char* e = strchr(seen_pointer, 'E');
+  char *e = strchr(seen_pointer, 'E');
   if (e) {
     *e = 0;
-    ret = strtod(seen_pointer + 1, NULL);
+    ret = strtod(seen_pointer+1, NULL);
     *e = 'E';
   }
   else
-    ret = strtod(seen_pointer + 1, NULL);
+    ret = strtod(seen_pointer+1, NULL);
```

Comment:
We tend to prefer spaces around operators in Marlin (even in such succinct cases).